main.js

```diff
@@ -85,6 +85,7 @@ app.on('ready', () => {
       contextIsolation: true,
       webPreferences: {
         enableRemoteModule: true,
+        contextIsolation: false,
```

Comment:
This was already turned off in our previous builds but the setting has changed to defaulting to true in new versions of electron thus requiring this explicit flag